panels/grafana-clickhouse-datasource/plugin.json

```diff
@@ -5,7 +5,7 @@
   "backend": true,
   "category": "sql",
   "dependencies": {
-    "grafanaDependency": "8.x || 9.x",
+    "grafanaDependency": "\u003e=9.5.0",
```

Comment:
For PMM2 we have `9.2.20`, are there any problems related to that?

Also, in regards to PMM3 does the version 10 have in impact on this datasource?

Reply:
No. Users on Grafana v9.x and higher of Grafana can use v4.

We have one Experimental dashboard using this datasource.